internal/controller/state/conditions/conditions.go

```diff
@@ -33,6 +33,10 @@ const (
 	// Route rules has a backendRef with an unsupported value.
 	RouteReasonBackendRefUnsupportedValue v1.RouteConditionReason = "UnsupportedValue"
 
+	// RouteReasonBackendRefUnsupportedProtocol is used with the "ResolvedRefs" condition when one of the
+	// Route rules has a backendRef with an unsupported protocol.
+	RouteReasonBackendRefUnsupportedProtocol v1.RouteConditionReason = "UnsupportedProtocol"
```

Comment:
This is already defined in the Gateway API, so we shouldn't need to define it ourselves.